changelog.txt

```diff
@@ -26,12 +26,13 @@ Changed
 - ``flow.directives`` is deprecated in favor of ``flow.FlowProject.operation.with_directives`` (#309, #502).
 - All environments require a scheduler in order to submit, even in pretend mode (#533).
 - Submitting in pretend mode will show additional scheduler command information (#533).
+- Support fractional timeout value in Pythonic command line interface (#541).
```

Comment:
I think this should be "Python", not "Pythonic", see also: https://www.computerhope.com/jargon/p/pythonic.htm

Reply:
@b-butler I cannot push to this branch, can you commit this suggestion?